providers/attio/metadata.go

```diff
@@ -0,0 +1,118 @@
+// nolint
+package attio
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/spyzhov/ajson"
+)
+
+// This struct is used for when the response data having slice of data.
+type responseObject struct {
+	Result []map[string]any `json:"data"`
+	// Other fields
+}
+
+var errCannotLoadMetadata = errors.New("cannot load metadata")
```

Comment:
There is a recent `common` error that could fit for use case: `ErrMissingExpectedValues`.
Otherwise, export this error, and maybe even not within connector but from `common` package.